app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CartsController < ApplicationController
+  def add
```

Comment:
By convention you 'should' name this route create.

Reply:
I'm not sure what you mean?

Reply:
Ah, now I read through it again. It should be called "create".

Reply:
I'm not sure I agree, but I'm not an expert on the conventions. :) 
Raoul was with us when we named it, but I think add_to_cart would be the best name to describe what we're doing.

Reply:
Not in the carts controller. But it's not a blocker. IMO we should stick to RESTful routes.

Reply:
Oh..... So the method name relates to the RESTful routes, and that's why it matters? I have no idea?

Reply:
We need resources :carts routes for that which we don't have or am I missing smth here?